tests/kafkatest/services/verifiable_producer.py

```diff
@@ -209,24 +182,16 @@ def start_cmd(self, node, idx):
             cmd += " --acks %s " % str(self.acks)
 
         cmd += " --producer.config %s" % VerifiableProducer.CONFIG_FILE
-        cmd += " 2>> %s 1>> %s &" % (VerifiableProducer.STDERR_CAPTURE, VerifiableProducer.STDOUT_CAPTURE)
+        cmd += " 2>> %s | tee -a %s &" % (VerifiableProducer.STDOUT_CAPTURE, VerifiableProducer.STDOUT_CAPTURE)
```

Comment:
I'm actually reverting this commit: 62e043a86565cc9bc485658d6c6d176e9aff620f which has STDOUT, STDOUT.
The original STDOUT, STDOUT was added here e6b343302f3208f7f6e0099fe2a7132ef9eaaafb without any notes, so it might just have been a typo.

It actually helps to have stdout (verifiable client json protocol) and stderr combined in the same log for troubleshooting, but I'm not saying it is right.

So in short: This does not add new behaviour, it reverts to the behaviour pre decemember 2016.

I can change it to stderr, stdout but that will require a full re-testing cycle of all my little clients so it will be a while.